src/plots/cartesian/dragbox.js

```diff
@@ -601,32 +601,35 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             subplots = Object.keys(plotinfos);
 
         for(var i = 0; i < subplots.length; i++) {
+
             var subplot = plotinfos[subplots[i]],
+                clipId = 'clip' + fullLayout._uid + subplots[i] + 'plot',
```

Comment:
Agreed. I'm always hesitant to add _more_ onto fullLayout, but I suppose the clips _do_ get used fairly often.